docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       - BIO=Hey! Just a place where you can connect with me!
       - GITHUB=https://github.com/timothystewart6
       - TWITTER=https://twitter.com/TechnoTimLive
-      - MASTODON=https://mastodon.social/@Gargron
+      - MASTODON=https://mastodon.social/TechnoTimLive
```

Comment:
any way you can just remove the line?

Reply:
Why removing the line ? 
It is needed to easily modify the target link in the docker-compose, no ?

Reply:
This is only the example.  All good.  I am going to rework examples soon.  Thank you!

Reply:
I just can not figure out your mastodon account so I take the maintainer's one.